packages/form-footer/src/FormFooter.tsx

```diff
@@ -178,8 +178,8 @@ export default function FormFooter({
               {cancelButtonText || 'Cancel'}
             </Button>
           )}
-          {isComponentType(primaryButton, 'Button') ? (
-            React.cloneElement(primaryButton, {
+          {isComponentType(primaryButton as React.ReactElement, 'Button') ? (
+            React.cloneElement(primaryButton as React.ReactElement, {
```

Comment:
Do we need this second assertion? the `isComponentType` should assert that it's a ReactNode, no?